src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -154,11 +155,12 @@ describe('EntryFieldsComponent', () => {
     component.setDataToUpdate(entry);
     spyOn(toastrServiceStub, 'error');
 
-    const hourInTheFuture = moment().add(1, 'hours').format('HH:mm:ss');
+    const hourInTheFuture = moment().add(1, 'hours').format('HH:mm');
     component.entryForm.patchValue({ start_hour : hourInTheFuture});
     component.onUpdateStartHour();
 
     expect(toastrServiceStub.error).toHaveBeenCalled();
+    expect(component.showtimeInbuttons).toEqual(false);
```

Comment:
typo, should be `component.showTimeInbuttons`

Reply:
I've already changed it, thanks.